test/unit/gateways/stripe_test.rb

```diff
@@ -1396,44 +1396,16 @@ def successful_partially_refunded_response(options = {})
   def successful_void_response
     <<-RESPONSE
     {
-      "id": "ch_4IrhQMqukqu7C2",
-      "object": "charge",
-      "created": 1403816613,
-      "livemode": false,
-      "paid": true,
-      "amount": 50,
+      "id": "re_173VMpAWOtgoysogOSE7Hzss",
```

Comment:
Stripe's AM adapter implements the `void` method using `/refund`, so wouldn't it be creating refunds?